bodhi/tests/client/__init__.py

```diff
@@ -892,4 +892,34 @@
   State:               pending
 """
 
+EXAMPLE_INFO_RELEASE_MUNCH = Munch({
+    u'dist_tag': u'f27', u'name': u'F27', u'testing_tag': u'f27-updates-testing',
+    u'pending_stable_tag': u'f27-updates-pending', u'pending_signing_tag': u'f27-signing-pending',
+    u'long_name': u'Fedora 27', u'state': u'pending', u'version': u'27',
+    u'override_tag': u'f27-override', u'branch': u'f27', u'id_prefix': u'FEDORA',
+    u'pending_testing_tag': u'f27-updates-testing-pending', u'stable_tag': u'f27-updates',
+    u'candidate_tag': u'f27-updates-candidate', u'critpath_pkgs': [
+        Munch({'stack_id': None, 'type': 'rpm', 'requirements': None,
+               'name': 'python', 'stack': None}),
+        Munch({'stack_id': None, 'type': 'rpm', 'requirements': None,
+               'name': 'kernel', 'stack': None})]})
+
+EXPECTED_INFO_RELEASE_OUTPUT = """Release:
+  Name:                F27
+  Long Name:           Fedora 27
+  Version:             27
+  Branch:              f27
+  ID Prefix:           FEDORA
+  Dist Tag:            f27
+  Stable Tag:          f27-updates
+  Testing Tag:         f27-updates-testing
+  Candidate Tag:       f27-updates-candidate
+  Pending Signing Tag: f27-signing-pending
+  Pending Testing Tag: f27-updates-testing-pending
+  Pending Stable Tag:  f27-updates-pending
+  Override Tag:        f27-override
+  State:               pending
+  Critpath Packages:   python,kernel
```

Comment:
Let's keep this as its own separate command so that the server doesn't have to fetch and serialize this data on every GET request for a release.